roles/openshift_health_checker/openshift_checks/diskspace_availability.py

```diff
@@ -0,0 +1,115 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class DiskAvailability(OpenShiftCheck):
+    """Check that recommended disk space is available."""
+
+    name = "disk_availability"
+    tags = ["preflight"]
+
+    recommended_node_space_gb = 15.0
+    recommended_master_space_gb = 40.0
+    minimum_docker_space_percent = 5.0
+
+    supported_docker_dev_driver = "devicemapper"
+
+    def run(self, tmp, task_vars):
+        self.recommended_node_space_gb = get_var(task_vars, "healthcheck_recommended_node_space_gb", default=self.recommended_node_space_gb)
+        self.recommended_master_space_gb = get_var(task_vars, "healthcheck_recommended_master_space_gb", default=self.recommended_master_space_gb)
+        self.minimum_docker_space_percent = get_var(task_vars, "healthcheck_minimum_docker_space_percent", default=self.minimum_docker_space_percent)
+
+        is_containerized = get_var(task_vars, "openshift", "common", "is_containerized")
+        ansible_mounts = get_var(task_vars, "ansible_mounts")
+
+        # if running containerized installation check that available
+        # space is not below defined minimum threshold
+        if is_containerized:
+            failed, msg = self.containerized_volume_check(task_vars)
+            if failed:
+                return {"failed": True, "msg": msg}
+
```

Comment:
The way it stands now, we will call both `containerized_volume_check` and `noncontainerized_volume_check` when `is_containerized` is true and `failed` is false...